### PR TITLE
Set-CsTenantFederationConfiguration - new AllowedTrialTenantDomains parameter docs

### PR DESCRIPTION
**Pull request by rhrivnak**
This PR contains documentation for new AllowedTrialTenantDomains parameter with examples of usage.

### CODE REVIEW

**Comment by rhrivnak** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Review by williamlooney — CHANGES_REQUESTED** [review]
Need to adjust/update wording for -AllowedTrialTenantDomains notes.

**Review by williamlooney — CHANGES_REQUESTED** [review]
Better, but, the text talks about what can be checked vs not.

Something like:

"Any domain in this list that belongs to a tenant with paid subscriptions will be ignored." us better.

File: teams/teams-ps/teams/Set-CsTenantFederationConfiguration.md [hunk 1]
```diff
@@ -475,6 +512,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowedTrialTenantDomains
+You can whitelist specific "trial-only" tenant domains, while keeping the `ExternalAccessWithTrialTenants` set to `Blocked`. This will allow you to protect your organization against majority of tenants that don't have any paid subscriptions, while still being able to collaborate externally with those trusted trial-tenants in the list. 
+ 
+Note: 
+- The list supports up to maximum 4k domains.
+- If `ExternalAccessWithTrialTenants` is set to `Allowed`, then the `AllowedTrialTenantDomains` list will not be checked. 
+- It's not possible to check whether an external domain has only trial subscriptions, when adding it to the `AllowedTrialTenantDomains` list. If a domain in the list doesn't belong to a tenant with only trial subscriptions, the system will automatically ignore it when enforcing the list.
```
[the remark this answers is not in the document]
Comment (rhrivnak) [reply teams/teams-ps/teams/Set-CsTenantFederationConfiguration.md]: Docs changed as requested.

**Review by williamlooney — COMMENTED** [review]
Approved

**Pull request merged**